hq/app/views/iam/iam.scala.html

```diff
@@ -3,11 +3,16 @@
 
 
 @main(List("IAM report")) { @* Header *@
-    <div class="container">
-        <div class="row">
-            <h1 class="header center-on-small-only grey-text text-lighten-5">IAM Report</h1>
-            <div class="col s12 m9">
-                <h4 class="grey-text text-darken-4 center-on-small-only">For all accounts</h4>
+    <div class="hq-sub-header">
+        <div class="container hq-sub-header__row">
+            <div class="hq-sub-header__name">
+                <h4 class="header light grey-text text-lighten-5">All accounts</h4>
+            </div>
+            <div class="hq-sub-header__tabs">
+                <ul class="tabs tabs-transparent">
+                    <li class="tab col s3"><a target="_self" href="/security-groups"><i class="material-icons left">vpn_lock</i>Security Groups</a></li>
+                    <li class="tab col s3"><a target="_self" class="active" href="/iam"><i class="material-icons left">vpn_key</i>IAM Report</a></li>
+                </ul>
```

Comment:
This code is duplicated with almost no variation in a few places. It would be good to split the sub-navigation out into a template fragment, so it can be reused rather than copy-pasted.

Will leave this for a future PR 😁 😉